.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'type:bug'
+assignees: ''
+
+---
+<!-- This template provide a checklist for reporting an issue for GCP Scanner. Modify the template if it's not suitable for your use case -->
+
+Your issue may already be reported! Please search on the [issue tracker](https://github.com/google/gcp_scanner/issues) before creating a new one.
+
+
+**Affected Component**
+<!-- Please tell us which component of GCP Scanner is affected (leave empty if you are unsure). -->
+
+* [ ] Crawl
+* [ ] CredsDB
+* [ ] Scanner
+* [ ] Test
+* [ ] Docs
+* [ ] Python
+* [ ] GCP APIs
+
+**Describe the bug**
+<!-- Please provide a clear and concise description of what the bug is. Code samples should be put in the **To Reproduce** section. -->
+
+**To Reproduce**
+<!-- Please provide detailed instructions on how to reproduce the behaviour, including code samples if applicable. -->
+
+Steps to reproduce the behavior:
```

Comment:
I'd probably show as an example something like run gcp_scanner with certain parameters

Reply:
I like it!